services/orchest-webserver/client/src/hooks/useCheckUpdate.tsx

```diff
@@ -178,7 +182,7 @@ export const useCheckUpdate = () => {
     // we want to be able to tell the user that no update is available
     // if this function is invoked.
     const [fetchedOrchestVersion, fetchedLatestVersion] = await makeCancelable(
-      requestToCheckVersions()
+      requestToCheckVersions(false)
```

Comment:
Building on the previous comment:

```suggestion
      requestToCheckVersions({ cache: false })
```